configs/setups/awicm3/awicm3.yaml

```diff
@@ -1,196 +1,657 @@
+
+        add_input_in_work:
+                areas: areas.nc
+                masks: masks.nc
+                grids: grids.nc
 
         add_restart_out_files:
                 rmp_a2f_B: rmp_a2f_B
-                rmp_a2f_D: rmp_a2f_D
-                rmp_f2a_D: rmp_f2a_D
+                rmp_f2a_G: rmp_f2a_G
+                rmp_a2r_B: rmp_a2r_B
+                rmp_r2f_G: rmp_r2f_G
 
         add_restart_out_in_work:
-                rmp_a2f_B: rmp_atmo_to_feom_BICUBIC.nc
-                rmp_a2f_D: rmp_atmo_to_feom_DISTWGT.nc
-                rmp_f2a_D: rmp_feom_to_atmo_DISTWGT.nc
+                rmp_a2f_B: ${fesom.nproc}/rmp_${oifs.oasis_grid_name_a}_to_feom_BICUBIC.nc
+                rmp_f2a_G: ${fesom.nproc}/rmp_feom_to_${oifs.oasis_grid_name_a}_GAUSWGT.nc
+                rmp_a2r_B: ${fesom.nproc}/rmp_${oifs.oasis_grid_name_r}_to_RnfA_BICUBIC.nc
+                rmp_r2f_G: ${fesom.nproc}/rmp_RnfO_to_feom_GAUSWGT.nc
 
         add_restart_out_sources:
-                rmp_a2f_B: rmp_atmo_to_feom_BICUBIC.nc
-                rmp_a2f_D: rmp_atmo_to_feom_DISTWGT.nc
-                rmp_f2a_D: rmp_feom_to_atmo_DISTWGT.nc
+                rmp_a2f_B: ${fesom.nproc}/rmp_${oifs.oasis_grid_name_a}_to_feom_BICUBIC.nc
+                rmp_f2a_G: ${fesom.nproc}/rmp_feom_to_${oifs.oasis_grid_name_a}_GAUSWGT.nc
+                rmp_a2r_B: ${fesom.nproc}/rmp_${oifs.oasis_grid_name_r}_to_RnfA_BICUBIC.nc
+                rmp_r2f_G: ${fesom.nproc}/rmp_RnfO_to_feom_GAUSWGT.nc
 
         add_restart_in_files:
                 rmp_a2f_B: rmp_a2f_B
-                rmp_a2f_D: rmp_a2f_D
-                rmp_f2a_D: rmp_f2a_D
+                rmp_f2a_G: rmp_f2a_G
+                rmp_a2r_B: rmp_a2r_B
+                rmp_r2f_G: rmp_r2f_G
 
         add_restart_in_in_work:
-                rmp_a2f_B: rmp_atmo_to_feom_BICUBIC.nc
-                rmp_a2f_D: rmp_atmo_to_feom_DISTWGT.nc
-                rmp_f2a_D: rmp_feom_to_atmo_DISTWGT.nc
+                rmp_a2f_B: ${fesom.nproc}/rmp_${oifs.oasis_grid_name_a}_to_feom_BICUBIC.nc
+                rmp_f2a_G: ${fesom.nproc}/rmp_feom_to_${oifs.oasis_grid_name_a}_GAUSWGT.nc
+                rmp_a2r_B: ${fesom.nproc}/rmp_${oifs.oasis_grid_name_r}_to_RnfA_BICUBIC.nc
+                rmp_r2f_G: ${fesom.nproc}/rmp_RnfO_to_feom_GAUSWGT.nc
 
         add_restart_in_sources:
-                rmp_a2f_B: rmp_atmo_to_feom_BICUBIC.nc
-                rmp_a2f_D: rmp_atmo_to_feom_DISTWGT.nc
-                rmp_f2a_D: rmp_feom_to_atmo_DISTWGT.nc
-
-
+                rmp_a2f_B: ${fesom.nproc}/rmp_${oifs.oasis_grid_name_a}_to_feom_BICUBIC.nc
+                rmp_f2a_G: ${fesom.nproc}/rmp_feom_to_${oifs.oasis_grid_name_a}_GAUSWGT.nc
+                rmp_a2r_B: ${fesom.nproc}/rmp_${oifs.oasis_grid_name_r}_to_RnfA_BICUBIC.nc
+                rmp_r2f_G: ${fesom.nproc}/rmp_RnfO_to_feom_GAUSWGT.nc
+
+
+# Set up CPU layout
+choose_cores_per_node:
+        24:
+                choose_resolution:
+                        T159_CORE2:
```

Comment:
Do we use these? If so they should be called TL or TCO + truncation number, rather than T + truncation number,

Reply:
Yes you are right. We should actually be careful to always use Tco, TL, or Tq
Changing it will probably break some code, but we need to be strict with this to avoid confusing, even if we are mostly using Tco.

Reply:
You are right @JanStreffing , this is effectively doing nothing as `resolution` contains TL or TCO so the case is never met. I will change the cases.

Reply:
Actually I will comment this whole `choose_` block. We are still a step behind, where we are trying to get a simulation running in Ollie with TL159_CORE2 truncation. We can uncomment it and generalize it later for offering support for other computers and truncations.